runtime/src/main/jni/JsV8InspectorClient.cpp

```diff
@@ -298,3 +296,4 @@ jclass JsV8InspectorClient::inspectorClass = nullptr;
 jmethodID JsV8InspectorClient::sendMethod = nullptr;
 jmethodID JsV8InspectorClient::sendToDevToolsConsoleMethod = nullptr;
 jmethodID JsV8InspectorClient::getInspectorMessageMethod = nullptr;
+int JsV8InspectorClient::contextGroupId = 1;
```

Comment:
It needs to be any non-zero number, making it a static just allows for easier changing, should any other v8 API require context grouping, and if we have to adjust the contextgroupId to be different.